### PR TITLE
Deno: improve the Readme file 

### PR DESCRIPTION
**Pull request by Rebaiahmed**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,29 +1,97 @@
-# Deno
+<h1 align="center">
+  <a name="logo" href="http://www.vCloudInfo.com/tag/iot"><img src="https://deno.land/logo.svg" alt="Deno Land" width="200"></a>
```
Comment (nayeemrmn) [inline README.md:2]: What's this URL?

Comment (Rebaiahmed) [reply README.md]: I just changed it

File: README.md [hunk 2]
```diff
@@ -1,29 +1,97 @@
...
+  <a name="logo" href="https://deno.land"><img src="https://deno.land/logo.svg" alt="Deno Land" width="200"></a>
+  <br>
+ A secure runtime for JavaScript and TypeScript 
+</h1>
+
+<div align="center">
+  <h4>
+    <a href="https://github.com/denoland/deno/actions"><img src="https://github.com/denoland/deno/workflows/ci/badge.svg?branch=master&event=push"/></a>
+    <a href="https://github.com/denoland/deno/stargazers"><img src="https://img.shields.io/github/stars/denoland/deno.svg?style=plasticr"/></a>
+    <a href="https://github.com/denoland/deno/commits/master"><img src="https://img.shields.io/github/last-commit/CCOSTAN/Home-AssistantConfig.svg?style=plasticr"/></a>
```
Comment (SyrupThinker) [inline README.md:11]: This is also not deno related

Comment (Rebaiahmed) [reply README.md:11]: > https://github.com/denoland/deno/blob/c7af17cc960d080a851ad1ffabd686161e2b1f6e/README.md
> 
> I'm sorry but these are bad changes. What are the table of contents and headers for? Most of the added badges provide info that is already visible on the main page.
What do you suggest ?

**Comment by Rebaiahmed** [comment]
> https://github.com/denoland/deno/blob/c7af17cc960d080a851ad1ffabd686161e2b1f6e/README.md
> 
> I'm sorry but these are bad changes. What is the table of contents and headers for? Most of the added badges provide info that is already visible on the main page.

As a user when i checked the Readme main file, it didn't' looks beautiful, I just tried to follow other open-source projects readme file, no section for contributors, all things are listed in links, what's the purpose of including links, why i didn't' find any information directly in the Readme file

**Comment by crowlKats** [comment]
> no section for contributors

that's bloat as there is the `contributors` section in `insights` tab

File: README.md (hunk 2 above)
Comment (SyrupThinker) [reply README.md:11]: Not sure what this comment has to do with this clearly unrelated URL:
`https://img.shields.io/github/last-commit/CCOSTAN/Home-AssistantConfig.svg`

**Comment by Rebaiahmed** [comment]
> that's bloat as there is the `contributors` section in `insights` tab

Where is it ? can you suggest a solution

**Comment by crowlKats** [comment]
> > that's bloat as there is the `contributors` section in `insights` tab
> 
> Where is it ? can you suggest a solution

Screenshot_2020-05-14 denoland deno

solution: not to include in the readme as it is on github itself.

**Comment by piscisaureus** [comment]
These are not the kind of changes we're looking for.

Plus it looks like you're trying to add spam links to our readme. If it was an honest mistake, make sure it was the last one of this kind.

**Pull request closed without merging**